app.js

```diff
@@ -33,8 +32,10 @@ app.use(require('express-session')({
 app.use(express.static(__dirname + '/public/static/'));
 app.use(express.static(__dirname + '/public/angular'));
 
+app.get(/^\/(index)?$/, function(req, res) {
+  res.sendFile(path.join(__dirname + '/views/index.html'));
+});
 app.use('/', staticPages);
-app.use('/', angularPage);
```

Comment:
Comment 2:
It's better to keep angularPage route instead of write route directly in app.js